assembl/static/js/app/views/message.js

```diff
@@ -1557,7 +1565,7 @@ var MessageView = Marionette.LayoutView.extend({
           i18n.gettext('Message has been successfully deleted.'),
           { delay: 12000 }
         );
-
+        that.fetchIdeasCollection();
         // Refresh the messageList
         that.messageListView.render();
         setTimeout(function(){
```

Comment:
Ah...yes, here it is. A setTimeout of 500ms. How come? We should make the `fetchIdeasCollection` promisified. And in the `then` method, have the messageList re-render and `showMessageById`.